specification/html/spec_apis.md

```diff
@@ -0,0 +1,122 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## APIs
+
+The API entity schema is used to define an OpenWhisk API within a manifest.
+
+### Fields
+<html>
+<table width="100%">
+ <tr>
+  <th width="16%">
+   <p>Key Name</p>
+  </th>
+  <th width="12%">
+   <p>Required</p>
+  </th>
+  <th width="16%">
+   <p>Value Type</p>
+  </th>
+  <td width="14%">
+   <p>Default</p>
+  </th>
+  <th width="40%">
+   <p>Description</p>
+  </th>
+ </tr>
+ <tr>
+  <td>method</td>
+  <td>yes</td>
+  <td>string</td>
+  <td>N/A</td>
+  <td>The HTTP method for the endpoint. All valid HTTP methods are supported, but a response type value of <b>http</b> may be required to correctly process the associated request.</td>
+ </tr>
+ <tr>
+  <td>response</td>
+  <td>no</td>
+  <td>string</td>
+  <td>http</td>
+  <td>The response type or <i>content extension</i> used when the API Gateway invokes the web action. See <a href="https://github.com/apache/incubator-openwhisk/blob/master/docs/webactions.md#content-extensions">https://github.com/apache/incubator-openwhisk/blob/master/docs/webactions.md#content-extensions</a>.</p></td>
+ </tr>
+</table>
+</html>
+
+### Grammar
+
+```yaml
+<apiName>:
+    <basePath>:
+        <relativePath>:
+            <actionName>:
+                method: <get | delete | put | post | ...>
+                response-type: <http | json | text | html>
```

Comment:
is this `response-type` or `response`? similarly few more places in the rest of the doc.

Reply:
it should be `response`.  i'll change all of these. :+1: